### ISSUE TITLE
Positions not updated properly on window resize

### ISSUE THREAD

**Issue by trueg** [alexfoxy/lax.js#108]
I am currently using lax to let a div float in from the side of the screen to land in the middle:

```
<div class="lax promo-body"
           data-lax-translate-x="vh 0, 0 (0.5*vw-0.5*elw), 1(0.5*vw-0.5*elw)"
           data-lax-translate-y="vh vh, 0 (0.5*vh-0.5*elh), 1 (0.5*vh-0.5*elh)"
           data-lax-anchor="self">
```

This works nicely but relies heavily on `elw` end the like.

Sadly when I resize the window `lax.updateElements()` does not correct the positioning. In fact one of the floats will be moved completely off screen.

Are my translate arguments too convoluted or am I mising something else?

**Comment by trueg** [comment]
I think I found the problem: when using `self` as an anchor the `anchorTop` value will not be calculated properly. I was able to "fix" it with the following addition after the calculation of `anchorTop`:

```
if (o["data-lax-anchor"] == el && o.el.style.hasOwnProperty('transform')) {
  o.anchorTop -= Math.floor(o.el.style.transform.replace(/.*translateY\(([\d\.]*)px\).*/g, '$1'));
}
```
The code is quite ugly. A better solution might involve remembering the translate values or sth like that. On top of that I did not test this "fix" with any of the presets yet. Hence no pull request.
